### PR TITLE
add data masjid

### PR DESCRIPTION
**Pull request by luqmanafif96**

### CODE REVIEW

File: app/data/institutions.ts [hunk 1]
```diff
@@ -7791,5 +7791,108 @@ export const institutions: Institution[] = [
 		qrContent: "00020201021126420014A000000615000101066033460210MD001289575204866153034585802MY5925MASJID NURUL ISLAM SG ITA6002MY627303251702546796248001928467473052017025468095260032574071617025460847970016304C7F6",
 		supportedPayment: ['duitnow', 'tng'],
 		coords: [6.41781084335408, 99.82726992574317],
-	}
+	},
+	
+	{
+		id: 680,
+		name: "Masjid Al-Zakirin Selayang ",
```
Comment (khrnchn) [inline app/data/institutions.ts:7798]:
```suggestion
		name: "Masjid Al-Zakirin Selayang",
```

File: app/data/institutions.ts [hunk 2]
```diff
@@ -7791,5 +7791,108 @@ export const institutions: Institution[] = [
...
+		state: "Selangor",
+		city: "Cheras",
+		qrImage: "",
+		qrContent: "00020201021126530014A000000615000101065641690221QRAMB00000000003754575204866153034585802MY5925MASJID BANDAR TASIK PUTER6008SELANGOR61054802062360317MASJID BTP RAWANG0711019-930004063043B8E",
+		supportedPayment: ['duitnow'],
+		coords: [3.291219330649202, 101.47465672642183],
+	  },
+	  {
+		id: 682,
+		name: "Masjid Dato Penghulu Menteri Sg Ujong ",
```
Comment (khrnchn) [inline app/data/institutions.ts:7820]:
```suggestion
		name: "Masjid Dato Penghulu Menteri Sg Ujong",
```

File: app/data/institutions.ts [hunk 3]
```diff
@@ -7791,5 +7791,108 @@ export const institutions: Institution[] = [
...
+		category: "mosque",
+		state: "Negeri Sembilan ",
```
Comment (khrnchn) [inline app/data/institutions.ts:7822]:
```suggestion
		state: "Negeri Sembilan",
```

File: app/data/institutions.ts [hunk 4]
```diff
@@ -7791,5 +7791,108 @@ export const institutions: Institution[] = [
...
+		city: " Nilai ",
```
Comment (khrnchn) [inline app/data/institutions.ts:7823]:
```suggestion
		city: " Nilai",
```

**Comment by khrnchn** [comment]
thank you broski :rocket:

**Comment by luqmanafif96** [comment]
wc

**Pull request merged**